### ISSUE TITLE
Feature request: Tell me what parts have/haven't been successfully nested

### ISSUE THREAD

**Issue by 1waggbri** [Jack000/Deepnest#137]
Currently the program tells me I have X number of parts nested out of Y. It would be really useful/helpful to know which files parts have/haven't been nested.

**Comment by JackS910** [comment]
Yes, I wanted this function too. I could then perhaps arrange the missing parts manually.
